src/main/java/uk/gov/ida/verifyserviceprovider/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -53,6 +49,10 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    public String getHubConnectorEntityId() {
```

Comment:
Could we either use Optional here or switch the other methods to do null checks in a consistent way?

Reply:
OK